packages/core/test/sdk.wrap.test.ts

```diff
@@ -0,0 +1,97 @@
+import { render } from '@testing-library/react-native';
+import * as React from 'react';
+
+import { wrap } from '../src/js/sdk';
+
+jest.mock('../src/js/touchevents', () => ({
+  __esModule: true,
```

Comment:
Is the ESM flag required in this case?

Reply:
it wasn't required, I have removed it from the tests on https://github.com/getsentry/sentry-react-native/pull/4740/commits/a38cbcb2b699583c1bcc8daf3a3efcac1aef825c 🙏